src/seedu/addressbook/data/AddressBook.java

```diff
@@ -1,9 +1,6 @@
 package seedu.addressbook.data;
 
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
Try to explicitly state imported classes, as stated [here](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html#package-and-import-statements).